### PR TITLE
Bps 641 populate document UUID values

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-641

### Change description ###
DB migration script to populate documentUuid values from documentUrl

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V041__Update_document_uuid_values.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Extracts the text after the last slash (/) in the documentUrl
+UPDATE scannable_items
+ SET documentUuid = REVERSE(SPLIT_PART(REVERSE(documentUrl), '/', 1))
```
Comment (rkondratowicz) [inline src/main/resources/db/migration/V041__Update_document_uuid_values.sql]: are both slots writing to that new column now?

Comment (aliveni) [reply src/main/resources/db/migration/V041__Update_document_uuid_values.sql]: the new column is in both slots. The previous pr (which writes the value to the new column) is merged but deployed once. the second build  is still in progress

Comment (nitinprabhu) [reply src/main/resources/db/migration/V041__Update_document_uuid_values.sql]: should we include a where clause here where documentuuid is null ?

Comment (aliveni) [reply src/main/resources/db/migration/V041__Update_document_uuid_values.sql]: added where clause

**Comment by aliveni** [comment]
Merged duplicate PR https://github.com/hmcts/bulk-scan-processor/pull/671

**Pull request closed without merging**